crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/builder/MaatCourtDataAssessmentBuilder.java

```diff
@@ -73,37 +78,50 @@ private MaatApiCreateAssessment buildCreate(MeansAssessmentRequestDTO requestDTO
     }
 
     private MaatApiUpdateAssessment buildUpdate(MeansAssessmentDTO assessment) {
         MeansAssessmentRequestDTO meansAssessment = assessment.getMeansAssessment();
 
         MaatApiUpdateAssessment updateAssessment = new MaatApiUpdateAssessment();
+        ApiUserSession userSession = meansAssessment.getUserSession();
 
         if (AssessmentType.FULL.equals(meansAssessment.getAssessmentType())) {
             updateAssessment
                     .withFullAscrId(assessment.getAssessmentCriteria().getId())
                     .withFassFullStatus(assessment.getCurrentStatus().getStatus())
                     .withFullAssessmentDate(meansAssessment.getFullAssessmentDate())
                     .withFullResult(ofNullable(assessment.getFullAssessmentResult())
-                            .map(FullAssessmentResult::getResult).orElse(null))
+                                            .map(FullAssessmentResult::getResult).orElse(null))
                     .withFullResultReason(ofNullable(assessment.getFullAssessmentResult())
-                            .map(FullAssessmentResult::getReason).orElse(null))
+                                                  .map(FullAssessmentResult::getReason).orElse(null))
                     .withFullAssessmentNotes(meansAssessment.getFullAssessmentNotes())
                     .withFullAdjustedLivingAllowance(assessment.getAdjustedLivingAllowance())
                     .withFullTotalAnnualDisposableIncome(assessment.getTotalAnnualDisposableIncome())
                     .withFullOtherHousingNote(meansAssessment.getOtherHousingNote())
-                    .withFullTotalAggregatedExpenses(assessment.getTotalAggregatedExpense());
+                    .withFullTotalAggregatedExpenses(assessment.getTotalAggregatedExpense())
+                    .withFinAssIncomeEvidences(
+                            mapIncomeEvidence(assessment.getMeansAssessment().getIncomeEvidence(), userSession));
         }
         return updateAssessment
-                .withUserModified(meansAssessment.getUserSession().getUserName())
+                .withUserModified(userSession.getUserName())
                 .withFinancialAssessmentId(meansAssessment.getFinancialAssessmentId());
     }
+
+    private List<FinancialAssessmentIncomeEvidence> mapIncomeEvidence(List<ApiIncomeEvidence> incomeEvidences,
+                                                                      ApiUserSession userSession) {
+        return incomeEvidences == null ? null : incomeEvidences.stream()
+                .map(item -> new FinancialAssessmentIncomeEvidence(
+                        item.getId(), item.getDateReceived(), item.getDateModified(), item.getActive(),
+                        item.getApiEvidenceType().getCode(), item.getMandatory(), item.getApplicantId(),
+                        item.getOtherText(), userSession.getUserName(), userSession.getUserName(), item.getAdhoc()
```

Comment:
We seems to be setting the userCreated and userModified to the same user here from the session. Are there some scenarios where a different user might update evidence to the user that created it? If this is the case then is this handled further up the chain in orchestrator perhaps?